docfx_project/ziti/quickstarts/network/common-quickstart.md

```diff
@@ -11,21 +11,20 @@ When first launched - the instace will deposit a file into the file system at
 Now, ssh to the newly created machine. Once there you can obtain the username and password for your
 Ziti Controller by issuing this command:
 
-    jq -r .password ~/.config/ziti/ziti-controller/credentials.json
+    jq . ~/.config/ziti/ziti-controller/credentials.json
 
-You can choose to keep this password or change it to something easier to remember. If you change the password, please
-remember to use a strong password which is not easy to guess.
+You can choose to keep this username and password or change it to something easier to remember. If you change the password, please remember to use a strong password which is not easy to guess.
 
 > [!TIP]
-> Once the password is changed - update the credentials.json file with the current password if you want the system to
-> automatically update the certificates in the event of the image losing its IP address or DNS entry.
+> If you change the server password via the UI then do also let the credentials.json reflect the new password if 

```

Comment:
change to:
If you change the server password update the credentials.json file. This ensures the system can automatically update the certificates (...)